Makefile

```diff
@@ -58,11 +58,7 @@ docs:
 	@echo '✓ Rebuild docs'
 	@$(WEBPACK)
 
-build: clean compile styles
-
-styles:
```

Comment:
so, we set this up a few years ago as a way to build a global stylesheet that apps could import  from CDN rather than going through the NPM package and it turned out to be a terrible idea - surprised it's stuck around this long, since we never really ended up using it 😂